qa/suites/ceph-deploy/basic/distros

```diff
@@ -1 +0,0 @@
-../../../distros/supported
```

Comment:
why do we want to remove RHEL from supported distros when running ceph-deploy tests?

Reply:
@tchaikov ceph-deploy and ceph-ansible work differently when rhel comes into picture(tools are configured with different assumptions)

eg: http://qa-proxy.ceph.com/teuthology/teuthology-2018-05-16_05:55:02-ceph-deploy-mimic-distro-basic-mira/2538001/teuthology.log

Reply:
```
2018-05-16T06:11:08.108 INFO:teuthology.orchestra.run.mira081.stderr:[mira081][INFO  ] Running command: sudo yum -y install ceph ceph-osd ceph-
mds ceph-mon ceph-radosgw
2018-05-16T06:11:08.377 INFO:teuthology.orchestra.run.mira081.stderr:[mira081][DEBUG ] Loaded plugins: fastestmirror, langpacks, priorities, pr
oduct-id, search-
2018-05-16T06:11:08.377 INFO:teuthology.orchestra.run.mira081.stderr:[mira081][DEBUG ]               : disabled-repos, subscription-manager
2018-05-16T06:11:08.641 INFO:teuthology.orchestra.run.mira081.stderr:[mira081][DEBUG ] Determining fastest mirrors
2018-05-16T06:11:08.706 INFO:teuthology.orchestra.run.mira081.stderr:[mira081][DEBUG ]  * epel: ftp.linux.ncsu.edu
2018-05-16T06:11:41.000 INFO:teuthology.orchestra.run.mira081.stderr:[mira081][DEBUG ] No package ceph available.
2018-05-16T06:11:42.719 INFO:teuthology.orchestra.run.mira081.stderr:[mira081][DEBUG ] No package ceph-osd available.
2018-05-16T06:11:44.589 INFO:teuthology.orchestra.run.mira081.stderr:[mira081][DEBUG ] No package ceph-mds available.
2018-05-16T06:11:46.308 INFO:teuthology.orchestra.run.mira081.stderr:[mira081][DEBUG ] No package ceph-mon available.
2018-05-16T06:11:48.178 INFO:teuthology.orchestra.run.mira081.stderr:[mira081][DEBUG ] No package ceph-radosgw available.
2018-05-16T06:11:49.145 INFO:teuthology.orchestra.run.mira081.stderr:[mira081][WARNING] Error: Nothing to do
2018-05-16T06:11:49.310 INFO:teuthology.orchestra.run.mira081.stderr:[mira081][ERROR ] RuntimeError: command returned non-zero exit status: 1
```

is it possible that we can setup a ceph repo for RHEL ?

Reply:
@tchaikov I am not sure its worth enhancing ceph-deploy to support rhel and there will be obviously more work on the build side too, unless someone is going to signup on the build/rpm items it would be wasted effort for little to no gain.

Reply:
> there will be obviously more work on the build side too

i think we are already building packages for RHEL, see http://docs.ceph.com/docs/master/install/get-packages/#add-ceph

by "setting up a ceph repo for RHEL", i mean we can update the yum settings so it can find the ceph packages.